src/flint/fmpq_rel_series.jl

```diff
@@ -900,6 +900,14 @@ function zero!(z::QQRelPowerSeriesRingElem)
   return z
 end
 
+function one!(z::QQRelPowerSeriesRingElem)
+  ccall((:fmpq_poly_one, libflint), Nothing,
+        (Ref{QQRelPowerSeriesRingElem},), z)
+  z.prec = parent(z).prec_max
+  z.val = parent(z).prec_max
```

Comment:
this seems wrong. Setting it to one would mean that the valuation is 0 (or I am thinking wrong)
```suggestion
  z.val = 0
```

Reply:
Yes you are right, this is wrong. Thank you!